client/src/components/HomePage.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const HomePage = (props) => {
+    return(
+        <div className="primary">
+
+            <h2 className="secondary header">Welcome to the Homepage</h2>
+        
+        <div>
```

Comment:
the div here and on line 14 should be removed as they do not add styling and only add clutter to the DOM with extra HTML elements